### PR TITLE
[PRM] LRSD-5579 - Enable debugging function on quickstart environment

### PR DESCRIPTION
**Pull request by abfranca**

### CODE REVIEW

**Comment by abfranca** [comment]
ci:test:sf

**Review by kylembischof — APPROVED** [review]

File: workspaces/liferay-partner-workspace/quickstart/docker-compose.yaml [hunk 1]
```diff
@@ -27,5 +27,6 @@ services:
         image: liferay/dxp:2024.q2.6
         ports:
             - 8080:8080
```
Comment (ryanschuhler) [inline workspaces/liferay-partner-workspace/quickstart/docker-compose.yaml:29]: @abfranca can you sort these to make the 8000 above 8080? other than that looks good!

File: workspaces/liferay-partner-workspace/quickstart/liferay.env [hunk 2]
```diff
@@ -10,6 +10,7 @@ LIFERAY_JDBC_PERIOD_DEFAULT_PERIOD_DRIVER_UPPERCASEC_LASS_UPPERCASEN_AME=com.mys
 LIFERAY_JDBC_PERIOD_DEFAULT_PERIOD_PASSWORD=
 LIFERAY_JDBC_PERIOD_DEFAULT_PERIOD_URL=jdbc:mysql://database/lportal?useUnicode=true&characterEncoding=UTF-8
 LIFERAY_JDBC_PERIOD_DEFAULT_PERIOD_USERNAME=root
+LIFERAY_JPDA_ENABLED=true
```
Comment (ryanschuhler) [inline workspaces/liferay-partner-workspace/quickstart/liferay.env:13]: What does this flag do for us? or what problem does it solve?

Comment (abfranca) [reply workspaces/liferay-partner-workspace/quickstart/liferay.env:13]: I've found it in this [slack thread](https://liferay.slack.com/archives/C01CSLH1UFJ/p1661350164526959).
Well, it worked without this other flag he mentioned `JPDA_ADDRESS=0.0.0.0:8000`, I can try with just the 8000 port exposed to see if this is really a requirement.

**Comment by ryanschuhler** [comment]
ci:forward

**Pull request closed without merging**